src/Microsoft.Diagnostics.Monitoring.EventPipe/Counters/CounterPipelineSettings.cs

```diff
@@ -8,18 +8,35 @@
 
 namespace Microsoft.Diagnostics.Monitoring.EventPipe
 {
-    internal class EventPipeCounterPipelineSettings : EventSourcePipelineSettings
+    internal class CounterPipelineSettings : EventSourcePipelineSettings
     {
         public EventPipeCounterGroup[] CounterGroups { get; set; }
```

Comment:
Long term, I think this should be cleaned up, but I don't want to make this change now. The api layers here are:

* Dotnet monitor configuration layer. Truly public api, no breaking changes.
* MetricsPipelineSettings/EventPipeCounterGroup represents configuration for our pipeline. That includes specific events we are interested in. Currently this is an internal api, but in the future it might become public. This has significant overlap with the layer below.
* MonitoringSourceConfiguration/MetricEventPipeProvider used for mapping settings to EventSource providers. Currently public (but not documented or anything). We do not need event names at this layer, since providers don't do that kind of filtering (the pipeline in the layer above does this).
* EventPipeProvider - NETCore.Client level configuration extracted from MonitoringSourceConfiguration.